File: src/Titlebar/Titlebar.cs
// Source code owned by: audiotore341
// 2022 License: GNU GENERAL PUBLIC LICENSE v2

using System;


namespace EchOS
{

    public class Titlebar
    {


        public static void DrawBar()
        {

            // X and y cords.
            int XCord = Console.CursorLeft;
            int YCord = Console.CursorTop;

            // Top right of the screen.
            Console.SetCursorPosition(0, 0);

            Console.ForegroundColor = ConsoleColor.White;
            Console.BackgroundColor = ConsoleColor.Red;

            var hour = Cosmos.HAL.RTC.Hour;
            var minute = Cosmos.HAL.RTC.Minute;
            var strhour = hour.ToString();
            var strmin = minute.ToString();

            Console.WriteLine("EchOS 1.5                        Build-12/25/2020                          " + strhour + ":" + strmin);

            if (Colorhandle.color == "Black")
            {

                Console.BackgroundColor = ConsoleColor.Black;

            }

            else if (Colorhandle.color == "Blue")
            {

                Console.BackgroundColor = ConsoleColor.Black;

            }

            else if (Colorhandle.color == "Yellow")
            {

                Console.BackgroundColor = ConsoleColor.Yellow;

            }

            else
            {
                Console.BackgroundColor = ConsoleColor.Black;
            }



            // Now reset cursor positon.

            Console.SetCursorPosition(XCord, YCord);
            Console.ForegroundColor = ConsoleColor.White;

        }

    }

}

File: src/EchOS/Titlebar.java
package EchOS;

public class Titlebar {
	
	public static final String ANSI_BLUE = "\u001B[34m"; // Blue color.
	public static final String ANSI_WHITE = "\u001B[37m";
	public static final String ANSI_BLUE_BACKGROUND = "\u001B[44m";
	public static final String ANSI_RESET = "\u001B[0m";
	
	public static void DrawBar() {
		
		char escCode = 0x1B;
		int row = 0;
		int column = 0;
		
		
		// Top right of the screen.
		System.out.print(String.format("%c[%d;%df",escCode,row,column));
		
		System.out.println(ANSI_BLUE_BACKGROUND + "                        Welcome! [EchOS 1.6] Kernel: 1.6                          " + ANSI_RESET);
		
		
		
	}
}